src/std/text/json/env.ss

```diff
@@ -1,33 +1,39 @@
 ;;; -*- Gerbil -*-
-;;; ̧© vyzo
+;;; © vyzo
 ;;; json io environment
 (import :std/error
         :std/sugar)
 (export #t)
 
-;; should decoded hashes have symbols as keys?
+;; Should decoded JSON "objects" have symbols as keys?
 (def json-symbolic-keys
   (make-parameter #t))
```

Comment:
Should we change this default too? It's a bigger change.

Reply:
If we're going to change defaults at all, better do it all at once than one setting at a time.

Reply:
yeah, I guess we should discuss it in gitter